app.js

```diff
@@ -23,8 +27,36 @@ const HEADLESS = (process.env.HEADLESS || "true").toLowerCase() === "true";
 const CONNECT_TIMEOUT = parseInt(process.env.CONNECT_TIMEOUT) || 180000;
 const VIEWPORT_WIDTH = parseInt(process.env.VIEWPORT_WIDTH) || 1280;
 const VIEWPORT_HEIGHT = parseInt(process.env.VIEWPORT_HEIGHT) || 720;
+const TWO_CAPTCHA_TOKEN = process.env.TWO_CAPTCHA_TOKEN || "0";
```

Comment:
Maybe we should name this 2CAPTCHA_TOKEN for easier mapping

Reply:
Variables cannot start with digit literal!